### PR TITLE
Here are the fixes for lassen

### PR DESCRIPTION
**Pull request by rdaly525**
The plan is to merge this branch, then merge the branches for the other repos, then revert these branches back to master.

### CODE REVIEW

**Comment by Kuree** [comment]
I will wait until peak and lassen are merged to master.

Ignore the travis build. It ran out of 8G memory.

**Comment by rdaly525** [comment]
Okay, I will merge lassen and peak.

**Comment by rdaly525** [comment]
Lassen and Peak are merged. Still need https://github.com/StanfordAHA/garnet/pull/359 merged

**Review by Kuree — APPROVED** [review]

**Pull request merged**
